notebooks/07_emg_hand_gesture_classification.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>NIT: let's try to use present tense:</p><p> </p><p>In these experiments, we want to ...</p><p> </p><p>Very NIT: You could add somewhere that "calibrating" means "training"? It could not be obvious for people (e.g. mathematicians) who have never heard of BCI.</p><p> </p><p>Transfert learning -&gt; transfer learning</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>